### ISSUE TITLE
Document purpose and how to run examples in Twisted Conch

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5590]
|<img alt="gdeng's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| gdeng reported|
|-|-|
|Trac ID|trac#5590|
|Type|enhancement|
|Created|2012-04-01 21:53:25Z|

Add documentation to some of the examples in /doc/conch/examples.

Attachments:

* [patch2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/063/063bcaf1e7942e73c9379ef6903d09fdd594d33f/db9e5b116ee90ce4e78ae3cd67cd388de4ae01c9.patch) (1771 bytes) - added by gdeng on 2012-04-01 23:45:09Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5590 5590
type__enhancement enhancement
reporter__gdeng gdeng
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1333317205000000 1333317205000000
changetime__1333890686000000 1333890686000000
version__None None
owner__itamar itamar
cc__jesstess cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="gdeng's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|gdeng removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst set owner to @itamarst|
|-|-|

Looks good, .tac files can really be mysterious to people without the explanation.

I will try merge this (probably after slightly reformatting the text to match our coding standard: http://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html#auto9) this evening after work, if someone else doesn't get to it first.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_4"></a>|@itamarst commented|
|-|-|

Oh, one last thing for future reference: since this is a conch change, the news file would need to go in `twisted/conch/topfiles/`.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_5"></a>|@itamarst set status to `closed`|
|-|-|

(In [34144]) Apply slightly tweaked patch: better docs for two conch demos.

Author: gdeng
Review: itamar
Fixes: [#5590](https://github.com/twisted/twisted/issues/5590)
